translations/story/50/1003/303 (デビュー戦の後に・帝王、発進！).json

```
@@ -1019,7 +1019,7 @@
             "jpName": "エアグルーヴ",
             "enName": "Air Groove",
             "jpText": "目指すのだろう？\r\n“クラシックレース”を。",
-            "enText": "I think they are aiming for a \"classic race\". \nThe \"Classic Race\".",
+            "enText": "You're aiming for it, right? \nThe \"Classic Race\".",
```

```suggestion
            "enText": "You're aiming for it, right? \nThe \"Classic Races\".",
```
Classic(s) refers either to 2nd year or the triple crown races which Teio aimed for.